### ISSUE TITLE
Screen Reader says "10 Billion" as "one zero billion"

### ISSUE THREAD

**Issue by stemilymill** [phetsims/gravity-force-lab-basics#301]
For https://github.com/phetsims/qa/issues/702

Test device : Dell laptop
Browser/screen reader combos:  Firefox & NVDA, Chrome and JAWS

When using the Change Mass spinner, if the mass is set to 10 billion kg, the screen reader says "one zero billion kilograms" instead of "ten billion kilograms."

This did not happen in the published version.

(I'm still learning terminology, please correct me so I can improve)

**Comment by jessegreenberg** [comment]
Thanks @stemilymill. Normally I would say this kind of screen reader output is out of our control but since it isn't happening in the published version maybe something has changed. Ill try to get it to happen myself and then look for anything that might have changed in the formatting of the value.

**Comment by jessegreenberg** [comment]
Confirmed with NVDA + Firefox.

**Comment by jessegreenberg** [comment]
@terracoda said that maybe in the published version we were spelling out numbers like "ten". Ill check into that. If we cannot find why this is happening or if something changed in the formatting @terracoda and @emily-phet suggested we could spell out the numbers.

**Comment by zepumph** [comment]
I confirmed in https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.3/phet/gravity-force-lab-basics_a11y_view.html that the published version is using real numbers, not words:

![image](https://user-images.githubusercontent.com/6856943/132555761-518f1938-8870-4ebb-a20c-a2b39c4ffe61.png)


There are the two items copied from each sim:

```html
<!--On master:-->
<input data-trail-id="388-389-563-570-565-734-735-709-706-705-691" id="388-389-563-570-565-734-735-709-706-705-691"
       data-focusable="true" class="a11y-pdom-element" type="range" aria-orientation="vertical"
       aria-valuetext="2 billion kilograms" min="1" max="10" step="1" aria-valuenow="2"
       aria-roledescription="number spinner">


<!--Published version:-->
<input data-trail-id="378-465-469-640-641-619-618-617-603" id="378-465-469-640-641-619-618-617-603"
       data-focusable="true" class="a11y-sibling" type="range" aria-orientation="vertical"
       aria-valuetext="3 billion kilograms" min="1000000000" max="10000000000" step="90000000"
       aria-valuenow="3000000000" aria-roledescription="number spinner">
```

**Comment by jessegreenberg** [comment]
Thanks @zepumph, thats helpful! I just confirmed that published version does not have this bug. The biggest difference I see is the min, max, and step attributes though I am not sure why that would cause this. Ill try to isolate in an example HTML case.

**Comment by jessegreenberg** [comment]
I verified that this is happening in the RC. I verified that this is NOT happening in the last published version. I have not been able to find a way to reproduce this in a basic HTML example. I tried adding attributes up to the slider that is in master. Then I tried copying the exact HTML in the PDOM into a different HTML page and didnt' see this happen. 

More surprising is that I just notice this is NOT happening in the unbuilt version off of master.

**Comment by jessegreenberg** [comment]
One difference I see between unbuilt and built is that in the built version there are directional embedding marks: 
![image](https://user-images.githubusercontent.com/6396244/133466905-4bd2f164-4336-42c8-abb5-f3870dde7339.png)

**Comment by jessegreenberg** [comment]
I just confirmed that removing the directional marks fixes this issue.

**Comment by jessegreenberg** [comment]
Resolved in the above commit. This has been cherry picked into the scenery gravity-force-lab-basics-1.1 release branch for GFL:B.

**Comment by KatieWoe** [comment]
Looks ok on Win 11
